sdcontent/INSTRUCTIONS-README.txt

```diff
@@ -14,21 +14,21 @@ NOOBS INSTALLATION INSTRUCTIONS
       iv. Check that the SD card you inserted matches the one selected by the Tool
       v. Click the "Format" button
    c. Linux
-      i. We recommend using gparted (or the command line version parted)
-      ii. Format the entire disk as FAT32
+      i. We recommend using gparted (or the command line version, parted)
+      ii. Format the entire card as FAT32
    d. Cards over 32GB
-      i. Follow the instructions on https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md
+      i. Follow the instructions at https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md
 3. Extract the files contained in this NOOBS zip file.
-4. Copy the extracted files onto the SD card that you just formatted so that this file is at the root directory of the SD card. Please note that in some cases it may extract the files into a folder, if this is the case then please copy across the files from inside the folder rather than the folder itself.
-5. Insert the SD card into your Pi and connect the power supply.
+4. Copy the extracted files onto the SD card that you just formatted. The files must be in the root directory of the SD card, not in a subdirectory within this. If the files are in a subdirectory on the SD card, then you will not be able to start a Raspberry Pi from the SD card.
+5. Insert the SD card into the Raspberry Pi and connect the power supply.
 
-**Raspberry Pi 4 Model B:** display must be connected to HDMI port closest to power jack.
+**Note:** on the Raspberry Pi 4B you must connect the display to the micro HDMI port closest to the power jack, labelled HDMI 0.
 
 Your Pi will now boot into NOOBS and should display a list of operating systems that you can choose to install.
 If your display remains blank, you should select the correct output mode for your display by pressing one of the following number keys on your keyboard:
 1. HDMI mode - this is the default display mode.
 2. HDMI safe mode - select this mode if you are using the HDMI connector and cannot see anything on screen when the Pi has booted.
-3. Composite PAL mode - select either this mode or composite NTSC mode if you are using the composite RCA video connector.
+3. Composite PAL mode - select either this mode or composite NTSC mode if you are using a screen connected to the composite video port. (Either RCA or 'headphone' connector, depending on model of Raspberry Pi).
 4. Composite NTSC mode
 
-If you are still having difficulties after following these instructions, then please visit the Raspberry Pi Forums ( http://www.raspberrypi.org/forums/ ) for support.
+If you are still having difficulties after following these instructions, you can ask for help on the Raspberry Pi forums at https://raspberrypi.org/forums.

```

Comment:
I think the `www` part of the URL ought to be kept, and perhaps it's worth putting a space before the full-stop to reduce the chances of the user trying to go to `https://www.raspberrypi.org/forums.` ?